public/less/_base.less

```diff
@@ -1,11 +1,85 @@
 @footer-margin: 40px;
 
+@default-fonts: "Segoe UI", "Microsoft YaHei", Arial, Helvetica, sans-serif;
+
+.override-fonts(@fonts) {
+    body {
+        font-family: "Helvetica Neue", Helvetica, "Segoe UI", Arial, @fonts;
+    }
+
+    .markdown:not(code) {
+        font-family: "Helvetica Neue", Helvetica, "Segoe UI", Arial, @fonts;
+    }
+
+    /* We're going to just override the semantic fonts here */
+    h1, h2, h3, h4, h5 {
+        font-family: "Lato", @fonts;
+    }
+
+    .home .hero h1, .home .hero h2 {
+        font-family: 'PT Sans Narrow', "Lato", @fonts;
+    }
+
+    .ui.accordion .title:not(.ui),
+    .ui.button,
+    .ui.card > .content > .header.ui.card > .content > .header,
+    .ui.category.search > .results .category > .name,
+    .ui.form input:not([type]), .ui.form input[type=date], .ui.form input[type=datetime-local], .ui.form input[type=email], .ui.form input[type=file], .ui.form input[type=number], .ui.form input[type=password], .ui.form input[type=search], .ui.form input[type=tel], .ui.form input[type=text], .ui.form input[type=time], .ui.form input[type=url],
+    .ui.header,
+    .ui.items > .item > .content > .header,
+    .ui.list .list > .item .header,
+    .ui.list > .item .header,
+    .ui.menu,
+    .ui.message .header,
+    .ui.modal > .header,
+    .ui.popup > .header,
+    .ui.search > .results .result .title,
+    .ui.search > .results > .message .header,
+    body,
+    .ui.input>input, .ui.input input,
+    .ui.statistics .statistic > .value,
+    .ui.statistic > .value,
+    .ui.statistics .statistic > .label,
+    .ui.statistic > .label,
+    .ui.steps .step .title,
+    .ui.text.container
+    {
+        font-family: "Lato", @fonts;
+    }
```

Comment:
What is this abomination of a selector list trying to do? Seems quite unmaintainable to me.

Reply:
That's to override semantic ui's font family